tests/tests-discounts.php

```diff
@@ -84,6 +84,11 @@ public function test_discounts_exists() {
 		$this->assertTrue( edd_has_active_discounts() );
 	}
 
+	public function test_is_discount_active() {
+		$this->assertTrue( edd_is_discount_active( $this->_post_id, true ) );
+		$this->assertTrue( edd_is_discount_active( $this->_post_id, false ) );
```

Comment:
This test needs to also verify that the status wasn't changed when we send in a `false`. So build one out with an expired discount, check that `is_active` with a `false` still returns it as `active` in the DB?